components/admin/DownloadBoostWinnersButton.tsx

```diff
@@ -28,9 +29,12 @@ const DownloadBoostWinnersButton: React.FC<DownloadBoostWinnersButtonProps> = ({
   };
 
   return (
-    <Button onClick={handleDownload}>
-      <p>Download boost winners</p>
-    </Button>
+    <DownloadButton 
+    label="Download Boost Winners" 
+    endpoint={`/api/boost-winners`} // Ensure the endpoint starts with /api/
```

Comment:
For the endpoint, it should not be a url, but a function. You should put () => AdminService.getBoostWinnersByBoostId({ id: Number(boostId) })